assets/airports/airportLoadList.js

```diff
@@ -263,6 +263,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
         //     name: 'Atatürk International Airport',
         //     wip: true
         // },
+        {
+            icao: 'mdsd',
+            level: 'easy',
+            name: 'Aeropuerto Internacional Las Américas',
+            wip: true
```

Comment:
Since all new airports will _have_ to be in full compliance before merge, when this does go in, it won't be a "work in progress". So for all new airports, and any pull requests aiming to bring an airport into the active list, you can go ahead and mark it as `wip: false` in the PR. :smiley: